File: Modul2/C/PRAK205-2310817210003-Muhammad_Azrianzan.c
#include <stdio.h>
#include <math.h>

int main() {
    int A;
    int B;
    printf("");
    scanf("%d %d", &A, &B);

    int alas = sqrt(pow(B, 2) - pow(A, 2));
    int tinggi = A;
    int keliling = A + B + alas;
    int luas = 0.5 * alas * tinggi;

    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);

    return 0;
}

File: Modul2/Python/PRAK205-2310817210003-Muhammad_Azrianzan.py
import math

A, B = input("").split()
A = int(A)
B = int(B)

alas = int(math.sqrt(math.pow(B, 2) - math.pow(A, 2)))
tinggi = A
keliling = int(A + B + alas)
luas = int(0.5 * alas * tinggi)

print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")